DEVELOPMENT.md

```diff
@@ -93,9 +93,9 @@ You may need to make changes to a pre-release prior to publishing a final stable
 - Once the release is published:
   - Pull the latest `release-*` branch containing the PR you just merged
   - Merge the `release-*` branch into `main` **using a non-fast-forward merge** and push it up to GitHub
-    - `git checkout main; git merge --no-ff release-next`
```

Comment:
Next in here is also not referring to the framework name

Reply:
we're trying to release-remix not release next js